docs/en/engines/table-engines/special/distributed.md

```diff
@@ -25,10 +25,19 @@ The Distributed engine accepts parameters:
     -   [insert_distributed_sync](../../../operations/settings/settings.md#insert_distributed_sync) setting
     -   [MergeTree](../../../engines/table-engines/mergetree-family/mergetree.md#table_engine-mergetree-multiple-volumes) for the examples
 
+Also it accept the following settings:
+
+- `fsync_after_insert` - Do fsync for every inserted. Will decreases performance of inserts (only for async INSERT, i.e. `insert_distributed_sync=false`),
+
+- `fsync_tmp_directory` - Do fsync for temporary directory (that is used for async INSERT only) after all part operations (writes, renames, etc.).
```

Comment:
```suggestion
- `fsync_after_insert` - do the `fsync` for the file data after async insert. Guarantees that the OS flushed the whole file to a disk **on the initiator node**. 
- `fsync_tmp_directory` - do the `fsync` for directories after async insert. Guarantees that the OS refreshed directory metadata **on the initiator node** disk.

!!! note "Note"
    Durability settings (`fsync_...`):
    - Affect only async INSERTs (i.e. `insert_distributed_sync=false`) when data first stored on the initiator node disk and later asynchronously send to shards.
    - May significantly decrease the inserts' performance
    - Affect writing the data stored inside Distributed table folder into the **node which accepted your insert**. If you need to have guarantees of writing data to underlying MergeTree tables - see durability settings (`...fsync...`) in `system.merge_tree_settings`.
```

Reply:
May be we should call the setting just `fsync_directories` ?

Reply:
Thanks for the suggestion, LGTM, although I did not want to include "OS" word into the description, but I guess it is fine (anyway not it looks way more understandable for users)

Reply:
>May be we should call the setting just fsync_directories or fsync_after_directory_operations ?

It was named by analogy to the MergeTree table, i.e. `fsync_part_directory`, I'm ok with both (actually even fsync_after_insert looks too verbose to me)

Reply:
> did not want to include "OS" word into the description

Initially i also was not using that, but grammarly recommended me to rephrase to remove passive voice ('guarantee part were flushed')

> I'm ok with both 

I like `fsync_directories`.